base_comment_template/tests/test_base_comment_template.py

```diff
@@ -87,3 +87,11 @@ def test_partner_template_domain(self):
         self.assertTrue(
             self.before_template_id not in self.partner_id.base_comment_template_ids
         )
+
+    def test_render_comment_text(self):
+        expected_text = "Test comment render %s" % self.user.name
+        self.before_template_id.text = "Test comment render ${object.name}"
```

Comment:
As far text can be translatable it could be nice to add a test that use a translated template to render a translatable field